dashboard/app/views/shared/_maybe_set_hoc_secret.html.haml

```diff
@@ -0,0 +1,12 @@
+- if current_user && current_user.admin?
+  - # For admins, set a client-side secret that allows them to access restricted HOC levels.
+  :javascript
+    //<![CDATA[
+    localStorage.setItem('hoc_secret', '#{DCDO.get("hoc_secret", nil)}');
```

Comment:
I added a try-catch to be safe rather than sorry.

(It shouldn't stop the other script blocks from running because it's a separate block, but didn't want to chance it.)